sound/soc/sof/ipc4-topology.h

```diff
@@ -349,12 +349,6 @@ struct sof_ipc4_process {
 	void *ipc_config_data;
 	uint32_t ipc_config_size;
 	struct sof_ipc4_msg msg;
-	/*
```

Comment:
@ujfalusi you missed one thing here. I think we should also remove the output_format field in this struct right?

Reply:
Actually @ujfalusi please don't remove the output_format. I think we will need it.